hadoop-common-project/hadoop-common/pom.xml

```diff
@@ -363,6 +363,10 @@
       <artifactId>wildfly-openssl-java</artifactId>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <groupId>org.xerial.snappy</groupId>
```

Comment:
<scope>provided</scope>

Reply:
We can make it provided, and once we create a `hadoop-compression` module, we can add back the jar. @viirya since the jar will be provided, we need to check if the class exists so we can log it with right message.